smartmodules/jolt/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "jolt"
+version = "0.0.0"
+authors = ["Fluvio Contributors <team@fluvio.io>"]
+description = "JSON to JSON transformation SmartModule"
+edition = "2021"
+license = "Apache-2.0"
+publish = false
+
+[lib]
+crate-type = ['cdylib', 'rlib']
+
+[dependencies]
+fluvio-jolt = { version = "0.1"}
+fluvio-smartmodule = { version = "0.3" }
```

Comment:
Should be using latest fluvio smartmodule on master until we release it

Reply:
It is using the latest from master. It's patched on workspace-level Cargo.toml

Reply:
I was wondering about that.  Great.